07-read-write.qmd

```diff
@@ -524,13 +524,11 @@ src
 
 All of the previous examples, like the one above, read spatial information from files stored on your hard drive.
 However, GDAL also allows reading data directly from online resources, such as HTTP/HTTPS/FTP web resources.
-The only thing we need to do is to add a `/vsicurl/` or `/vsicurl_streaming/` (see [here](https://gis.stackexchange.com/questions/450105/using-rasterio-to-read-ndvi-data-from-gimmss-cog-products)) prefix before the path to the file.
```

Comment:
vsicurl_streaming isn't as useful as it seems. People new to geocomputing with Python shouldn't be advised to use it.

Reply:
For example, it doesn't support efficient slicing (windows) or access to overviews in COGs.